docker-compose.yml

```diff
@@ -161,21 +161,23 @@ services:
       KAFKA_LDAP_JAVA_NAMING_FACTORY_INITIAL: com.sun.jndi.ldap.LdapCtxFactory
       KAFKA_LDAP_COM_SUN_JNDI_LDAP_READ_TIMEOUT: 3000
       KAFKA_LDAP_JAVA_NAMING_PROVIDER_URL: ldap://openldap:389
-      # How to authenticate to LDAP
+      # Authenticate to LDAP
       KAFKA_LDAP_JAVA_NAMING_SECURITY_PRINCIPAL: cn=admin,dc=confluentdemo,dc=io
       KAFKA_LDAP_JAVA_NAMING_SECURITY_CREDENTIALS: admin
       KAFKA_LDAP_JAVA_NAMING_SECURITY_AUTHENTICATION: simple
-      # How to locate users and groups
-      KAFKA_LDAP_SEARCH_MODE: USERS
+      # Locate LDAP users and groups
+      KAFKA_LDAP_SEARCH_MODE: GROUPS
+      KAFKA_LDAP_GROUP_SEARCH_BASE: ou=groups,dc=confluentdemo,dc=io
+      KAFKA_LDAP_GROUP_NAME_ATTRIBUTE: cn
+      KAFKA_LDAP_GROUP_OBJECT_CLASS: group
```

Comment:
I think this line could be removed due to L174 is correct one?

Reply:
Looks like the same issue below on lines 306/308 as well with duplicate settings for `KAFKA_LDAP_GROUP_OBJECT_CLASS`

Reply:
@jeffhuang26 good catch!  I will delete this line.